.env.schema

```diff
@@ -3,5 +3,10 @@ REACT_APP_API=https://...
 REACT_APP_FACEBOOK_APP_ID=
 REACT_APP_GOOGLE_APP_ID=
 
+REACT_APP_DEFAULT_REDIRECT_URI=https://...
+
+# comma separated list of valid redirect uris
+REACT_APP_OTHER_REDIRECT_URIS=https://localhost:3001,https://localhost:3000
```

Comment:
Can the addresses contain wildcards? Would be great if we could do`https://*--kf-auth-stories.netlify.com`

Reply:
this env variable is poorly named. It's a list of the urls that don't need the redirect. So currently that would only be `http://localhost:3001` and `https://kf-auth-stories.netlify.com`